Client/Frontend/Browser/HomePanel/NewTabPageBackgroundDataSource.swift

```diff
@@ -124,7 +116,6 @@ class NewTabPageBackgroundDataSource {
             let attemptSponsored = Preferences.NewTabPage.backgroundSponsoredImages.value
                 && backgroundRotationCounter == NewTabPageBackgroundDataSource.sponsorshipShowValue
                 && !PrivateBrowsingManager.shared.isPrivateBrowsing
-                && BraveAds.isCurrentLocaleSupported()
```

Comment:
Client's shouldn't be checking for this, this is up to the server.

Reply:
Sounds good to me